CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.7.1
```

Comment:
```suggestion
## Unreleased
```

This allows us to bundle multiple changes into a release (which could be `1.8.0` or even `2.0.0`).